participants/Cox/pr_tutorial/buggy_function.py

```diff
@@ -30,3 +33,6 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
 
     format_string = '{}:{}:{:.' + str(decimals) + 'f}'
     return format_string.format(hours, minutes, seconds)
+
+# print the output of the code to check
+print(angle_to_sexigesimal(RA_deg))
```

Comment:
no need to test code in function, every time the function is imported it will run the print function at the bottom!

Reply:
Fixed in new push